rockcraft.yaml

```diff
@@ -8,38 +8,36 @@ base: ubuntu:22.04
 platforms:
   amd64:
 
+run_user: _daemon_
+
+environment:
+  SPARK_HOME: /opt/spark
+  SPARK_CONFS: /etc/spark8t/conf
+  JAVA_HOME: /usr/lib/jvm/java-11-openjdk-amd64
+  PYTHONPATH: /opt/spark/python:/opt/spark8t/python/dist:/usr/lib/python3/dist-packages
+  PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/spark:/opt/spark/bin:/opt/spark/python/bin:/opt/spark-client/python/bin
+  HOME: /home/spark
+  KUBECONFIG: /home/spark/.kube/config
+  SPARK_USER_DATA: /home/spark
+  SPARK_LOG_DIR: /var/log/spark
+
 services:
   entrypoint:
     command: "/bin/bash /opt/pebble/charmed-spark-entrypoint.sh"
```

Comment:
I'm thinking we should avoid using `bash` for our own scripts in favour of `sh` unless there's some compelling reason to use `bash`